pages.nl/common/gnmic-subscribe.md

```diff
@@ -0,0 +1,24 @@
+# gnmic subscribe
+
+> Abonneer op een gnmic netwerk apparaat staat updates.
```

Comment:
What was the English sentence? The sentence doesn't feel right.

Reply:
This is the English sentence: ` Subscribe to a gnmic network device state updates.`

Reply:
```suggestion
> Abonneer op een gnmic netwerk apparaat status updates
```

Maybe this sounds better. What is your opinion @sebastiaanspeck?